MantisPhpClient.php

```diff
@@ -10,60 +10,66 @@
  * A php client for MantisBT / MantisHub SOAP API.
  */
 class MantisPhpClient {
+    const CLIENT_VERSION = '1.0.1';
+
     /**
-     * The Soap Client.
+     * @var SoapClient The Soap Client.
      */
     private $soap_client;
 
     /**
-     * The uri for the webservice.
+     * @var string The uri for the webservice.
      */
     private $soap_uri;
 
     /**
-     * The uri for MantisBT instance.
+     * @var string The uri for MantisBT instance.
      */
     private $mantis_uri;
 
     /**
-     * The MantisBT version.
+     * @var string The MantisBT version.
      */
     private $mantis_version;
 
     /**
-     * The cached list of projects.
+     * @var array The cached list of projects.
      */
     private $projects;
 
     /**
-     * The user name.
+     * @var string The user name.
      */
     private $username;
 
     /**
-     * The user password.
+     * @var string The user password.
      */
     private $password;
 
     /**
-     * Username returned by mc_login() this should be used for further queries to webservice.
-     * It will be different from $username in case of anonymous login where username will be
-     * blank and this one will be the actual username.
+     * @var string Username returned by mc_login() this should be used for further queries to webservice.
+     *              It will be different from $username in case of anonymous login where username will be
+     *              blank and this one will be the actual username.
      */
     private $effective_username;
 
     /**
-     * The constructor.
+     * MantisPhpClient constructor.
+     * @param string $p_soap_wsdl_url
+     * @param string$p_username
+     * @param string $p_password
+     * @param string string $p_user_agent
      */
     public function __construct( $p_soap_wsdl_url, $p_username, $p_password, $p_user_agent = 'MantisPhpClient' ) {
```

Comment:
Let's default $p_user_agent to null here.  Then inside the constructor if value is null, set it to MantisPhpClient/CLIENT_VERSION rather than just MantisPhpClient.